yoti-sdk/src/main/java/com/yoti/mobile/android/sdk/YotiSDKButton.java

```diff
@@ -72,6 +87,10 @@ public void setOnYotiAppNotInstalledListener(@Nullable OnYotiAppNotInstalledList
         mOnYotiAppNotInstalledListener = listener;
     }
 
+    public void setOnYotiCalledListener(@NonNull OnYotiCalledListener listener) {
```

Comment:
Don't think this should be `@NonNull`; we handle null when we use it and setting to null may be required to prevent leaks

Reply:
Done, thanks 👍